src/builtin_pwd.c

```diff
@@ -0,0 +1,12 @@
+#include "bash_ops.h"
+#include "env_access.h"
+#include "ft_printf.h"
+
+int	builtin_pwd(t_command *cmd)
+{
+	char	*current;
```

Comment:
current? cur_dir, pwd or anything else. current is what runs your phone

don't assume because it's a small function that it won't be expanded at some point

other than that, check validity of returned value before calling ft_dprintf(), return -1 on failure